Libraries/Utilities/Appearance.js

```diff
@@ -85,6 +85,22 @@ module.exports = {
     return nativeColorScheme;
   },
 
+  setColorScheme(colorScheme: ?ColorSchemeName): void {
+    const nativeColorScheme: ?string =
+      colorScheme == null ? 'unspecified' : colorScheme;
+
+    invariant(
+      nativeColorScheme === 'dark' ||
+        nativeColorScheme === 'light' ||
+        nativeColorScheme == null,
+      "Unrecognized color scheme. Did you mean 'dark', 'light' or null?",
+    );
```

Comment:
```suggestion
    invariant(
      colorScheme === 'dark' ||
        colorScheme === 'light' ||
        colorScheme == null,
      "Unrecognized color scheme. Did you mean 'dark', 'light' or null?",
    );
```

At this point `nativeColorScheme` should never be null, since we replaced that with "unspecified". We could instead maybe add this invariant before we derive nativeColorScheme, earlier in the function. E.g. the below, but moved up more: